tests/vlan/test_autostate_disabled.py

```diff
@@ -112,6 +113,18 @@ def shutdown_multiple_with_confirm(self, duthost, interfaces, err_handler=loggin
         if not wait_until(60, 5, 0, self.check_interface_oper_state, duthost, interfaces, "down"):
             err_handler('shutdown "{interfaces}" failed'.format(interfaces=interfaces))
 
+        for interface in interfaces:
+            config_entry = json.loads("""
+                [{
+                    "PORT":{
+                        "%s":{
+                            "admin_status": "down"
+                        }
+                    }
+                }]
+            """ % interface)
+            delete_running_config(config_entry, duthost)
```

Comment:
Can the deletion configurations for all interfaces are combined into single configuration?

Reply:
done